RecoLocalCalo/HcalRecProducers/python/HcalSimpleReconstructor_hf_cfi.py

```diff
@@ -1,6 +1,6 @@
 import FWCore.ParameterSet.Config as cms
 
-hfreco = cms.EDProducer("HcalSimpleReconstructor",
+hfsimplereco = cms.EDProducer("HcalSimpleReconstructor",
     correctionPhaseNS = cms.double(0.0),
```

Comment:
as mentioned in https://github.com/cms-sw/cmssw/pull/21289/files#r152877625
the content of this file becomes redundant and you can either remove it and replace instances made out of this file with clones of  RecoLocalCalo.HcalRecProducers.hosimplereco_cfi.hosimplereco
or simply change the content of this file with 
``` hfreco =  RecoLocalCalo.HcalRecProducers.hosimplereco_cfi.hosimplereco.clone() ```
To minimize redundant cloning, it's better to remove this file.
A lazier option, to minimize changes elsewhere, would preserve this file.